src/zzz_od/gui/app_setting/notorious_hunt_setting_interface.py

```
@@ -291,3 +332,40 @@ def _on_order_changed(self, new_data_list: list[ChargePlanItem]) -> None:
+            tab_name='训练',
+            category_name='恶名狩猎',
+            mission_type_name='初生死路屠夫',
+            mission_name=None,
+            level='默认等级',
+            auto_battle_config='全配队通用',
+            run_times=0,
+            plan_times=1,
+            predefined_team_idx=-1,
+        )
+        self.card = NotoriousHuntCard(self.ctx, idx=-1, plan=self.plan)
+        # 对话框内仅配置计划，隐藏置顶与删除控件
```

Ruff (0.15.14)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)